charts/OpenSDS Installation using Helm.md

```diff
@@ -0,0 +1,334 @@
+# OpenSDS Installation using Helm # 
+
+## Prerequisite ##
+
+### Ubuntu
+version information:
+```bash
+root@proxy:~# cat /etc/issue
+Ubuntu 16.04.2 LTS \n \l
+```
+
+### Environment packages
+```bash
+apt-get update && apt-get install -y gcc make libc6-dev socat gcc make libc-dev
+apt-get install -y git curl wget libltdl7 libseccomp2 python-pip 
+```
+
+### Install Go (v1.12.1)
+```bash
+wget https://storage.googleapis.com/golang/go1.12.1.linux-amd64.tar.gz
+tar -C /usr/local -zxvf go1.12.1.linux-amd64.tar.gz   
+echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile  
+echo 'export GOPATH=$HOME/gopath' >> /etc/profile  
+source /etc/profile
+```
+
+### Docker
+version information:
+```bash
+root@ubuntu:~# docker version
+Client:
+ Version:           18.06.1-ce
+ API version:       1.38
+ Go version:        go1.10.3
+ Git commit:        e68fc7a
+ Built:             Tue Aug 21 17:24:56 2018
+ OS/Arch:           linux/amd64
+ Experimental:      false
+
+Server:
+ Engine:
+  Version:          18.06.1-ce
+  API version:      1.38 (minimum version 1.12)
+  Go version:       go1.10.3
+  Git commit:       e68fc7a
+  Built:            Tue Aug 21 17:23:21 2018
+  OS/Arch:          linux/amd64
+  Experimental:     false
+```
+
+You can install docker using these commands below:
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb 
+```
+
+### [Kubernetes](https://github.com/kubernetes/kubernetes) local cluster
+You can startup `v1.14.0` k8s local cluster by executing commands blow:
+```bash
+cd $HOME && git clone https://github.com/kubernetes/kubernetes.git
+cd $HOME/kubernetes && git checkout v1.14.0
+make
+echo alias kubectl='$HOME/kubernetes/cluster/kubectl.sh' >> /etc/profile
+hack/install-etcd.sh
+ENABLE_DAEMON=true ALLOW_PRIVILEGED=true FEATURE_GATES=VolumeSnapshotDataSource=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh -O
+```
+
+Note: If there is some DNS problems with your Kubernetes cluster, please refer to [here](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/) for debugging resolution.
+
+### Helm Installation ###
+* This will fetch the latest version of helm :
+```
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get >get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh
+helm init
+```
+
+### Keystone installation and configuration ###
```

Comment:
If the keystone is enabled in `opensds/value.yaml`, there is no need for this step.

Reply:
I thought we have to install keystone separately, I will remove this, Thanks.